cmd/ocm-support/find/registryCredentials/cmd.go

```diff
@@ -1,4 +1,4 @@
-package show
+package registry_credentials
```

Comment:
following the ocm-cli, I they don't seem to use dashes or camel case for composed words. Should we do the same?

https://github.com/openshift-online/ocm-cli/blob/main/cmd/ocm/list/machinepool/cmd.go

We would have to change the folder names too.